torchio/transforms/augmentation/intensity/random_gamma.py

```diff
@@ -0,0 +1,56 @@
+from typing import Tuple, Optional, List
+import torch
+from ....torchio import DATA, TypeRangeFloat
+from ....data.subject import Subject
+from .. import RandomTransform
+
+
+class RandomGamma(RandomTransform):
+    r"""Change contrast of an image by setting its values to the power
+    :math:`\gamma`.
+
+    Args:
+        log_gamma_range: Tuple :math:`(a, b)` to compute the factor
```

Comment:
We haven't used `_range` in other transforms where a range is expected.

I feel like passing the range of log makes more sense than a standard range, but can you please explain this?

Reply:
I think it's mainly a design choice. Again it's related to the `..note` but to avoid numerical instability, one needs that `x ** gamma` does not go towards infinity, this is ensured only if x is in [0, 1] and `gamma >= 0`. Therefore passing a range of log allows to always have `gamma > 0` (`gamma = 0` is not interesting).

Reply:
I understand! I meant that we haven't used the `parameter_range` syntax in other transforms. For example, in `RandomAffine` we use `scales` and `degrees`, although ranges are also accepted.

Reply:
Yes I don't know why I have written `log_gamma_range` instead of `log_gamma`, I will remove `_range`.